code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -220,7 +220,7 @@
 
 /obj/item/reagent_containers/food/drinks/bottle/bottleofnothing
 	name = "Bottle of Nothing"
-	desc = "A bottle filled with nothing."
+	desc = "A bottle filled with nothing, but probably something."
```

Comment:
Just delete this whole drink lol, we don't have mimes.

Reply:
elites told me not to delete anything on the codebase, im just making it more friendly for scp13

Reply:
sorry, deleting this breaks things- someone else can pr the deletion.